### DISCUSSION TITLE
Checked Exception과 Unchecked Exception의 차이점이 무엇인가요?

### DISCUSSION THREAD

**Discussion by P-C-Space** [NHN-Team-03/ITP#12] [General]
Exception의 발생시 이 둘은 무엇이며, 이 둘의 차이점은 무엇입니까?

**Comment by f1v3-dev** [comment]
Checked Exception: RuntimeException 상속하지 않음, try...catch문으로 예외 처리를 무조건 해야 한다. (throw가 되는 상황을 고려)
Uncheked Exception: RuntimeException을 상속, 명시적으로 예외처리를 하지 않아도 된다.

**Reply by P-C-Space** [reply]
Checked Exception은 Exception을 바로 상속

**Comment by P-C-Space** [comment]
그렇다면 Error와 Exceptio의 차이는 무엇입니까?

**Reply by Huni0819** [reply]
Error : 사용자가 예측할 수 없게 발생하는 문제. 예측하지 못하므로 처리가 불가능합니다.
Exception : 사용자가 예측할 수 있는 수준의 문제. 예측할 수 있으므로 try-catch문을 통해서 처리가 가능합니다.
